src/robot/libdocpkg/model.py

```diff
@@ -17,6 +17,7 @@
 
 from robot.model import Tags
 from robot.utils import getshortdoc, Sortable, setter
+from robot.utils.table_of_content import add_toc
```

Comment:
I don't think this Libdoc related util belongs to `robot.utils`. If it was to be added there, then it should be exposed directly from `robot.utils`, not from a submodule. Also, Python module names in general shouldn't use `_`.

It would be best to move that `table_of_content` module directly under `libdocpkg` and rename it to `tocbuilder` or something like that.